validator.php

```diff
@@ -2,7 +2,11 @@
 
 // validates that all security advisories are valid
 
-require __DIR__.'/vendor/autoload.php';
+if (!is_file($autoloader = __DIR__.'/vendor/autoload.php')) {
```

Comment:
yeah but this repo is for any kind of php user who maybe is not aware of composer.

the contrib readme states the following:

> Make sure your file validates by running php validator.php from the root of this project.

It does not mention composer at all. It should be either there in the readme or here. Both would be maximum friendly to newbies.